### ISSUE TITLE
[YARN] Can not install @bryntum product packages using yarn v2/v3

### ISSUE THREAD

**Issue by SergeyMaltsev** [bryntum/support#5719]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=44&t=23152&p=114502#p114502)

## Yarn v.2

```
yarn add @bryntum/scheduler@npm:@bryntum/scheduler-trial@5.2.4
➤ YN0000: ┌ Resolution step
➤ YN0001: │ TypeError: @bryntum/scheduler-trial-lib@npm:1.0.0: Invalid Version: latest
```

Everything is ok when I install @bryntum/scheduler-angular. The package is installed successfully.

System info:

```
yarn --version 2.4.3
node --version v16.15.0
```

## Yarn v.3

```
"YN0001: │ Error: @bryntum/calendar@npm:@bryntum/calendar-trial isn't supported by any available resolver"
```

Related [forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=26300&p=131076#p131076)

**Comment by danielpza** [comment]
Changing the postinstall scripts seems to work:

```diff
// package.json:
   "scripts": {
-    "postinstall": "node ../gantt-lib/build.js"
+    "postinstall": "node postinstall.js"
   },
```

```js
// postinstall.js
require("@bryntum/gantt-lib/build.js")
```

**Comment by SergeyMaltsev** [comment]
@danielpza 
Thanks for sharing this info. We will check if that helps with yarn package manager.
Seems that Yarn creates temporary folder structure during installing packages, which differs from existing in regular npm package manager,  so is can not reach  `../gantt-lib/build.js`

**Comment by danielpza** [comment]
To clarify, the issue is with [Yarn PnP](https://yarnpkg.com/features/pnp). It works fine with yarn node modules linker on v2/v3:

```
yarn config set nodeLinker node-modules
```

**Comment by danielpza** [comment]
Thanks @SergeyMaltsev, looking forward for a fix :-)

**Comment by SergeyMaltsev** [comment]
@danielpza so if someone configures yarn with that `nodeLinker` setting it could be a workaround for the installation bug without changing our packages?

**Comment by danielpza** [comment]
Yarn PnP is the default strategy but can be disabled. Yes it is a workaround for using it with yarn v2... 
Still yarn PnP has some advantages hopefully bryntum can support it

**Comment by danielpza** [comment]
For more information about the different install modes with yarn v2 see https://yarnpkg.com/features/linkers

**Comment by SergeyMaltsev** [comment]
@danielpza
Thanks for your feedback! 
Yes, we will check your proposal for fixing packages. 
We need this solution to work with regular npm PM also.

**Comment by lassesteffen** [comment]
That would be super cool! Are there any workarounds?

**Comment by SergeyMaltsev** [comment]
@lassesteffen does it have same problems as in yarn?
Could you please provide some more information?

**Comment by SergeyMaltsev** [comment]
@lassesteffen 
Thank you for the information.
We have ready development build which works for `yarn install` or `yarn add` commands.
We plan to release it in coming patch release for `5.6.x`

**Comment by SergeyMaltsev** [comment]
We have released `5.6.1` with support for yarn. Please try it and share your results.
